clip_image_classifier.py

```diff
@@ -1,15 +1,15 @@
 from jina import Executor, DocumentArray, requests
-from numpy.core.fromnumeric import argmax
 from transformers import CLIPProcessor, CLIPModel
-from typing import List
+from typing import List, Optional
 import torch
+from jina.logging.logger import JinaLogger
 
 class CLIPImageClassifier(Executor):
     """Classifies an image given a list of text classes using CLIP model"""
 
     def __init__(
         self,
-        classes: List[str] = ["this is a person","this is a cat","this is a dog"],
+        classes: Optional[List[str]] = None,
```

Comment:
I think we generally have some defaults in our executors, to make sure things work out of the box.

Reply:
Can this [vocab file](https://huggingface.co/openai/clip-vit-base-patch32/blob/main/vocab.json) be used as to fill the default `classes` args? @azayz

Reply:
It's a good idea but these are words used for tokenization, to my knowledge classes should be phrases describing what the image is about